algorithms/src/snark/varuna/ahp/indexer/circuit.rs

```diff
@@ -126,31 +133,22 @@ impl<F: PrimeField, MM: SNARKMode> Circuit<F, MM> {
         crate::fft::EvaluationDomain::<F>::new(self.index_info.num_variables).unwrap().size()
     }
 
-    /// Iterate over the indexed polynomials.
-    pub fn iter(&self) -> impl Iterator<Item = &LabeledPolynomial<F>> {
-        // Alphabetical order
-        [
-            &self.a_arith.col,
-            &self.b_arith.col,
-            &self.c_arith.col,
-            &self.a_arith.row,
-            &self.b_arith.row,
-            &self.c_arith.row,
-            &self.a_arith.row_col,
-            &self.b_arith.row_col,
-            &self.c_arith.row_col,
-            &self.a_arith.row_col_val,
-            &self.b_arith.row_col_val,
-            &self.c_arith.row_col_val,
-        ]
-        .into_iter()
+    pub fn arithmetize_polys(&self) -> impl Iterator<Item = LabeledPolynomial<F>> {
```

Comment:
Maybe we should find a better name for this; the polynomials are already the arithmetized versions

Reply:
Done 68f294cea